### PR TITLE
[#168] Add static app to host AlphaLyrae font

### PR DESCRIPTION
**Pull request by bglownia**

### CODE REVIEW

File: apps/static/src/environments/environment.prod.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const environment = {
```
Comment (dexturr) [inline apps/static/src/environments/environment.prod.ts:1]: can we remove this file?

File: apps/static/src/environments/environment.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// This file can be replaced during build by using the `fileReplacements` array.
```
Comment (dexturr) [inline apps/static/src/environments/environment.ts:1]: can we remove this file?

File: apps/static/src/app/app.element.ts [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+
+  connectedCallback() {
+    const title = 'static';
+    this.innerHTML = `
+    <div class="wrapper">
+      <div class="container">
+        <!--  WELCOME  -->
+        <div id="welcome">
+          <h1>
+            <span> Hello there, </span>
```
Comment (mattrussell36) [inline apps/static/src/app/app.element.ts:14]: Can we clear out all this NX boilerplate code?

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: apps/static/src/app/app.element.ts (hunk 3 above)
Comment (bglownia) [reply apps/static/src/app/app.element.ts]: I removed all references to it and just forgot to remove files itself. It's cleaned up

**Review by dexturr — APPROVED** [review]

**Pull request merged**